backend/apps/gateway/test/app.e2e-spec.ts

```diff
@@ -17,7 +44,619 @@ describe('AppController (e2e)', () => {
     await app.init();
   });
 
-  it('/auth/google (GET)', () => {
-    return request(app.getHttpServer()).get('/auth/google').expect(302);
+  // Generate access tokens for mock users
+  let MOCK_ADMIN_TOKEN: string;
+  let MOCK_USER_1_TOKEN: string;
+  let MOCK_USER_1_REFRESH_TOKEN: string;
+  beforeEach(async () => {
+    const userServiceClient: ClientGrpc = app.get(Service.USER_SERVICE);
+    const userAuthService = userServiceClient.getService<UserAuthServiceClient>(
+      USER_AUTH_SERVICE_NAME,
+    );
+    const { accessToken: adminAccessToken, refreshToken: adminRefreshToken } =
+      await firstValueFrom(userAuthService.generateJwts(MOCK_ADMIN_USER));
+    const { accessToken: user1Token, refreshToken: user1RefreshToken } =
+      await firstValueFrom(userAuthService.generateJwts(MOCK_USER_1));
+    MOCK_ADMIN_TOKEN = adminAccessToken;
+    MOCK_USER_1_TOKEN = user1Token;
+    MOCK_USER_1_REFRESH_TOKEN = user1RefreshToken;
+  });
+
+  describe('/auth endpoints', () => {
```

Comment:
Can we split up this file into multiple e2e files for better readability and maintenance?